pkg/doc/verifiable/credential.go

```diff
@@ -421,9 +603,80 @@ func loadCredentialSchema(url string, client *http.Client) ([]byte, error) {
 	return gotBody, nil
 }
 
-// JSONBytes converts Verifiable Credential to JSON bytes
-func (vc *Credential) JSONBytes() ([]byte, error) {
-	rawCred := &rawCredential{
+// JWT serializes Credential to signed JWT.
+func (vc *Credential) JWT(
+	signatureAlg JWTAlgorithm,
+	privateKey interface{},
+	keyID string,
+	minimizeVc bool) (string, error) { // nolint:lll
+
+	// currently jwt encoding supports only single subject (by the spec)
+	jwtClaims := &jwt.Claims{
+		Issuer:    vc.Issuer.ID,                   // iss
+		NotBefore: jwt.NewNumericDate(*vc.Issued), // nbf
+		ID:        vc.ID,                          // jti
+		Subject:   vc.getFirstSubjectID(),         // sub
+		IssuedAt:  jwt.NewNumericDate(*vc.Issued), // iat (not in spec, follow the interop project approach)
+	}
+	if vc.Expired != nil {
+		jwtClaims.Expiry = jwt.NewNumericDate(*vc.Expired) // exp
+	}
+
+	var raw *rawCredential
+	if minimizeVc {
+		vcCopy := *vc
+		vcCopy.Expired = nil
```

Comment:
This claim is missing from `jwtClaims` above

Reply:
It's set between, as `Expired` field of Verifiable Credential is optional and could be nil. See lines 621-623.